eternalcore-core/src/main/java/com/eternalcode/core/feature/serverlinks/package-info.java

```diff
@@ -0,0 +1,6 @@
+package com.eternalcode.core.feature.serverlinks;
+
+
+/*
+This feature work's only with Minecraft version 1.21 or newer
```

Comment:
I don't know it is useful because we don't generate javadocs for the core module. However we currently use @FeatureDocs annotation to generate json for own docs